### PR TITLE
GH-1924: Document "long" Turtle/TriG format variant

### PR DESCRIPTION
**Pull request by rybesh**
See https://github.com/apache/jena/pull/1925

### CODE REVIEW

**Review by kinow — APPROVED** [review]

File: source/documentation/io/rdf-output.md
Comment (afs) [inline source/documentation/io/rdf-output.md:293]: Its also possible to write
```
  RDFWriter.source(model)
    .format(RDFFormat.TURTLE_LONG)
    .output(System.out);
```

**Review by afs — APPROVED** [review]

**Comment by afs** [comment]
Thanks!

For features that only apply after a release, we leave the PR open/approved and merge when the version with the feature in it is released.

File: source/documentation/io/rdf-output.md
Comment (rybesh) [reply source/documentation/io/rdf-output.md:299]: I added your example.

**Pull request merged**
